src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -0,0 +1,360 @@
+                  >
+                    Save Alert
+                  </Button>
+                </Flex>
+              </Media>
+
+              <Media lessThan="md">
+                <Spacer mt={4} />
+
+                <Button
+                  loading={isSubmitting}
+                  width="100%"
+                  onClick={() => handleSubmit()}
+                  disabled={isSaveAlertButtonDisabled}
+                >
+                  Save Alert
+                </Button>
+
+                <Spacer mt={1} />
+
+                <Button
+                  variant="secondaryOutline"
+                  width="100%"
+                  onClick={onDeleteClick}
+                >
+                  Delete Alert
+                </Button>
+              </Media>
+            </Join>
+          </Box>
+        )
+      }}
+    </Formik>
+  )
+}
+
+export const SavedSearchAlertEditFormFragmentContainer = createFragmentContainer(
+  SavedSearchAlertEditForm,
+  {
+    savedSearch: graphql`
+      fragment SavedSearchAlertEditForm_savedSearch on SearchCriteria {
+        internalID
+        acquireable
+        additionalGeneIDs
+        artistIDs
+        atAuction
+        attributionClass
+        colors
+        dimensionRange
+        sizes
+        height
+        inquireableOnly
+        internalID
+        locationCities
+        majorPeriods
+        materialsTerms
+        offerable
+        partnerIDs
+        priceRange
+        userAlertSettings {
+          name
+          email
+          push
+        }
+        width
+      }
+    `,
+    artist: graphql`
+      fragment SavedSearchAlertEditForm_artist on Artist {
+        internalID
+        name
+        slug
+      }
+    `,
+    artworksConnection: graphql`
+      fragment SavedSearchAlertEditForm_artworksConnection on FilterArtworksConnection {
+        aggregations {
+          slice
+          counts {
+            count
+            name
+            value
+          }
+        }
+      }
+    `,
+  }
+)
+
+const SAVED_SEARCH_ALERT_EDIT_FORM_QUERY = graphql`
+  query SavedSearchAlertEditFormQuery($id: ID!, $artistId: String!) {
+    me {
+      savedSearch(id: $id) {
+        ...SavedSearchAlertEditForm_savedSearch
+      }
+    }
+    artist(id: $artistId) {
+      ...SavedSearchAlertEditForm_artist
+    }
+    artworksConnection(
+      first: 0
+      artistID: $artistId
+      aggregations: [
+        ARTIST
+        LOCATION_CITY
+        MATERIALS_TERMS
+        MEDIUM
+        PARTNER
+        COLOR
+      ]
+    ) {
+      ...SavedSearchAlertEditForm_artworksConnection
+    }
+  }
+`
+
+export const SavedSearchAlertEditFormQueryRenderer: React.FC<SavedSearchAlertEditFormQueryRendererProps> = ({
+  editAlertEntity,
+  onDeleteClick,
+  onCompleted,
+}) => {
+  return (
+    <SystemQueryRenderer<SavedSearchAlertEditFormQuery>
+      query={SAVED_SEARCH_ALERT_EDIT_FORM_QUERY}
+      variables={{ id: editAlertEntity.id, artistId: editAlertEntity.artistId }}
```

Comment:
**suggestion:** I think it might be better to use `artistIds` here instead of `artistId` because of [this](https://github.com/artsy/metaphysics/blob/e8db28b995098509c73e1f2785e74cf6fcb42eb7/src/data/gravity.graphql#L1920-L1930)
```suggestion
      variables={{ id: editAlertEntity.id, artistIds: [editAlertEntity.artistId] }}
```

Reply:
Here we need to pass `artistId` for [this](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L306) and [this](https://github.com/artsy/force/blob/8d6e5277b254d4aa70cb7428c5986b4e56b21e82/src/v2/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx#L311)

Reply:
oh good call, I confused it with the `SearchCriteriaAttributes` input